client/src/components/portfolio/Portfolio.vue

```diff
@@ -12,18 +12,20 @@
         <p>{{ assetPercent }}</p>
       </div>
     </v-card>
-    <v-card class="value-card" id="chart-container">
```

Comment:
We might want to use a more specific name for the id here, or we could use a class instead. I just think we'll likely end up having more than one chart-container